### DISCUSSION TITLE
如何声明一个已知地址的 PublicKey type 参数？

### DISCUSSION THREAD

**Discussion by linghuccc** [openbuildxyz/mina-zkapp-bootcamp#229] [Q&A] [answered]
const publicKey1 = new PublicKey(
  'B62qjJaXMmZgaNecUUrDZ384uDQGYAAoTRTX7CAQ1YrBT6yo3gbzCCJ'
);

根据 o1js 文档，mina 地址也就是 public key，属于 PublicKey type 参数。我试着用上面的代码声明自己的 mina 地址，但是收到了下面的错误：

![error](https://github.com/user-attachments/assets/0c9d836d-b71b-4b6a-8892-49d2118eca33)

求助高手

**Comment by logere** [comment, marked as answer]
这个地址是 Base58 编码, 需要用 `PublicKey.fromBase58(address)` 转换成 PublicKey

**Comment by linghuccc** [comment]
已回复
